.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
-env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+matrix:
+  include:
+    - python: 2.7
+      env: 
+        - TOXENV=flake8
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27-marionette
+    - python: 3.5
+      env: TOXENV=flake8
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35-marionette
+  exclude:
+    - python: 2.7
+    - python: 3.5
+
+  allow_failures:
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35-marionette
 install:
-- pip install tox
+  - pip install tox
```

Comment:
I put back tox-travis but it now fails. I require more time to understand how to make it work. But to be honest, it lookde fine without it.